Sources/ComposableArchitecture/Effects/Concurrency.swift

```diff
@@ -102,5 +102,21 @@ import SwiftUI
       }
       .eraseToEffect()
     }
+
+    /// Creates an effect that executes some work in the real world that doesn't need to feed data
+    /// back into the store.
+    ///
+    /// - Parameters:
+    ///   - priority: Priority of the underlying task. If `nil`, the priority will come from
+    ///     `Task.currentPriority`.
+    ///   - work: A closure encapsulating some work to execute in the real world.
+    /// - Returns: An effect.
+    public static func fireAndForget(
+      priority: TaskPriority? = nil,
+      _ work: @escaping @Sendable () async -> Void
+    ) -> Effect {
+      Effect<Void, Never>.task { await work() }
```

Comment:
Didn't you forget to pass the priority argument to `Effect.task`?
```suggestion
      Effect<Void, Never>.task(priority: priority) { await work() }
```

Reply:
🤦‍♂️ thank you!